test/runtests.jl

```diff
@@ -161,4 +161,78 @@ ans = [2.94429906542  1.16946130841  1.99831775701;
 zi = evalgrid(spl, xi, yi)
 @test zi ≈ ans
 
+# -----------------------------------------------------------------------------
+# Test 2D integration
+
+f_test = [] #initialize an empyt list in which I will store the functions to test
+
+# Functions to test
+function test2D_1(x::Float64, y::Float64)
+  1 - x^2 -y^2
+end
+
+function test2D_2(x::Float64, y::Float64)
+  cos(x) + sin(y)
+end
+
+function test2D_3(x::Float64, y::Float64)
+  x*exp(x-y)
+end
+
+push!(f_test, test2D_1)
+push!(f_test, test2D_2)
+push!(f_test, test2D_3)
```

Comment:
Should just be able to write
```
f_test = [test2D_1, test2D_2, test2D_3]
```
instead of using `push!`. (Similar with `range_2D_integration` below.) Also, the Julia convention is to name functions all lower-case and types camel-case. So, `2d` rather than `2D`.